diamm/views/website/archive.py

```diff
@@ -13,6 +13,14 @@ class ArchiveList(generics.ListAPIView):
     renderer_classes = (HTMLRenderer, renderers.JSONRenderer)
     pagination_class = ObjectPagination
 
+    def initial(self, request, *args, **kwargs):
+        letter = request.GET.get('l', None)
+
+        if letter:
+            self.queryset = self.queryset.filter(name__istartswith=letter)
+
+        super(ArchiveList, self).initial(request, args, kwargs)
```

Comment:
Oops... Yeah, it should be

On Jul 13, 2016, at 6:35 PM, Andrew Hankinson <notifications@github.com<mailto:notifications@github.com>> wrote:


In diamm/views/website/archive.py<https://github.com/diamm/diamm/pull/100#discussion_r70721104>:

> @@ -13,6 +13,14 @@ class ArchiveList(generics.ListAPIView):
>      renderer_classes = (HTMLRenderer, renderers.JSONRenderer)
>      pagination_class = ObjectPagination
>
> +    def initial(self, request, *args, **kwargs):
> +        letter = request.GET.get('l', None)
> +
> +        if letter:
> +            self.queryset = self.queryset.filter(name__istartswith=letter)
> +
> +        super(ArchiveList, self).initial(request, args, kwargs)


Shouldn't this be request, *args, **kwargs?

-
You are receiving this because you were assigned.
Reply to this email directly, view it on GitHub<https://github.com/diamm/diamm/pull/100/files/e348fa84976ad8a8cbc16c149e45a84e755cd2d6..b1d5d1caafa2c5228c684bd2a81cc4bc17791a38#r70721104>, or mute the thread<https://github.com/notifications/unsubscribe/AH8h-VxgVKCRtYVUf3qVGMKz4PGfDow6ks5qVWg1gaJpZM4JCeCe>.